packages/gestalt/src/TextArea.css

```diff
@@ -10,3 +10,36 @@
 .textArea::placeholder {
   color: var(--g-colorGray200);
 }
+
+.unstyledTextArea, .textAreaSpacer {
+  composes: unstyled from "./FormElement.css";
+  composes: Text fontSize3 from "./Text.css";
+  line-height: 1.5;
```

Comment:
i wonder if there are implications with i18n. we shouldnt apply line-height  because some languages are super tall. check with thai etc.

Reply:
Changes not allowed
Do not use the following CSS style properties:
line-height: Property in CSS that controls the space between lines of text. Not defining a line-height allows the browser to determine line-height based on language which prevents localization issues.

Reply:
This is to match the line height of `.textArea` on line 5

Reply:
OK,  then this is a Gestalt issue ahahah no worries.